src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const payment = 40;
+  let sum = days * payment;
+
+  if (days > 6) {
+    sum -= 50;
+  } else if (days > 2) {
```

Comment:
4. [Checklist] Prefer `if` with `return` over `if else` to simplify later conditions.